src/Persisters/Serializers/Native.php

```diff
@@ -3,6 +3,9 @@
 namespace Rubix\ML\Persisters\Serializers;
 
 use Rubix\ML\Persistable;
+use RuntimeException;
+use function get_class;
+use function is_object;
```

Comment:
Classy function imports, nice touch, we generally put a space between the last class import and the first function import (same with imported constants). Check the Travis logs for the messages coming from cs-fixer.

https://travis-ci.org/github/RubixML/RubixML/builds/691331649